demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -1,22 +1,231 @@
+    complience.create_incident_field('emailaddress', {'id': 'incident_emailaddress', 'name': 'E-mail Address'})
+
+    cortex_xdr = repo.create_pack('CortexXDR')
+    cortex_xdr.create_classifier('Cortex XDR - IR', {'id': 'Cortex XDR - IR', 'name': 'Cortex XDR - IR',
+                                                     'transformer': '', 'keyTypeMap': {}, 'type': 'classification'})
+
+    impossible_traveler = repo.create_pack('ImpossibleTraveler')
+    impossible_traveler.create_script('CalculateGeoDistance').create_default_script('CalculateGeoDistance')
+    impossible_traveler.create_playbook('Impossible_Traveler').create_default_playbook('Impossible Traveler')
+    impossible_traveler.create_test_playbook(
+        'playbook-Impossible_Traveler_-_Test').create_default_test_playbook('Impossible Traveler - Test')
+    impossible_traveler.create_layout('Impossible_Traveler', {"TypeName": "Impossible Traveler", "kind": "details",
+                                                              "layout": {}, "typeId": "Impossible Traveler"})
+    impossible_traveler.create_incident_field('Coordinates', {'id': 'incident_coordinates', 'name': 'Coordinates'})
+    impossible_traveler.create_incident_field('Previous_Coordinates', {'id': 'incident_previouscoordinates',
+                                                                       'name': 'Previous Coordinates',
+                                                                       "associatedTypes": ["Impossible Traveler"]})
+    impossible_traveler.create_incident_field('previouscountry', {'id': 'incident_previouscountry',
+                                                                  'name': 'previouscountry'})
+    impossible_traveler.create_incident_field('Previous_Sign_In_Date_Time',
+                                              {'id': 'incident_previoussignindatetime',
+                                               'name': 'Previous Sign In Date Time'})
+    impossible_traveler.create_incident_field('Previous_Source_IP', {'id': 'incident_previoussourceip',
+                                                                     'name': 'Previous Source IP'})
+    impossible_traveler.create_incident_field('Sign_In_Date_Time', {'id': 'incident_signindatetime',
+                                                                    'name': 'Sign In Date Time'})
+    impossible_traveler.create_incident_field('Travel_Map_Link', {'id': 'incident_travelmaplink',
+                                                                  'name': 'Travel Map Link'})
+    impossible_traveler.create_incident_type('Impossible_Traveler', {'id': 'impossibletraveler',
+                                                                     'name': 'Impossible Traveler',
+                                                                     "playbookId": "Impossible Traveler",
+                                                                     'preProcessingScript': '', 'color': 'test'})
```

Comment:
This creation is very long, please consider splitting it to helper methods ( for each pack) or at least order in sections